### ISSUE TITLE
certain features should fail instead of returning nonsense values

### ISSUE THREAD

**Issue by tmchartrand** [AllenInstitute/ipfx#390]
In the case of input_resistance and fi_fit_slope, negative values are biologically meaningless and indicate noise or other issues with the data. Although they can be filtered out in analysis, I think it would be better to record a failure for that feature.

This is a low priority fix that I'll get a PR together for at some point if it sounds reasonable, but as this would depend on #278, I'll wait until that's resolved first.
